test/backquote_code.html

```diff
@@ -0,0 +1,4 @@
+<p>before pre</p>
+<pre><code class="python">print("hello")</code>

```

Comment:
Is it normal or sort of standard to use `class` attribute to mark the language of the code ?
Any example that I can find on internet that follow such rule ?

Reply:
I find some websites in [this link](
https://publicwww.com/websites/%22%3Cpre%3E%3Ccode+class%3D%5C%22python%5C%22%22/)

Reply:
Hmm. I'm most used to seeing this convention on GitHub and it doesn't appear to follow this convention. For example, inspect the HTML of the following.

```python
print("hello")
```

The HTML is:

```
<div class="highlight highlight-source-python"><pre><span class="pl-c1">print</span>(<span class="pl-s"><span class="pl-pds">"</span>hello<span class="pl-pds">"</span></span>)</pre></div>
```

I like the idea of the backquotes, but perhaps we should leave out the language name until we can identify a more consistent convention.